server/models/contact.coffee

```diff
@@ -28,6 +28,14 @@ module.exports = class Contact extends cozydb.CozyModel
         # vCard Name = splitted
         # (Familly;Given;Middle;Prefix;Suffix)
         n             : String
+        org           : String
+        title         : String
+        department    : String
+        bday          : String
+        nickname      : String
+        url           : String
+        import        : Boolean
+        rev           : Date
```

Comment:
I need to override the save function of CozyDb to ensure that rev is updated each time a change occurs. What do you recommend me to do about it?